test/controllers/income/estimatedPay/update/IncomeUpdateCalculatorControllerSpec.scala

```diff
@@ -614,23 +938,15 @@ class IncomeUpdateCalculatorControllerSpec
   "taxablePayslipAmountPage" must {
     "display taxablePayslipAmount page" when {
       "journey cache returns employment name, id and payPeriod" in {
-        val testController = createTestIncomeUpdateCalculatorController
-        val cachedAmount = Some("9888787")
-        val payPeriod = Some(MONTHLY)
 
-        val mandatoryKeys = Seq(UpdateIncome_IdKey, UpdateIncome_NameKey)
-        val optionalKeys = Seq(UpdateIncome_PayPeriodKey, UpdateIncome_OtherInDaysKey, UpdateIncome_TaxablePayKey)
+        implicit val request = RequestBuilder.buildFakeRequestWithAuth("GET")
```

Comment:
Why is this marked implicit?

Reply:
Because the taxablePayslipAmount (expectedViewModel) requires an implict request and already has all of its other requirements implicitly.